frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx

```diff
@@ -71,13 +71,28 @@ const filterSelected = (list: BackingStoreStateType[], tableId: number) => {
   return list.filter((e) => e.selectedBy === tableId || e.selectedBy === '').sort(sort);
 };
 
-const getTableRows = (list: K8sResourceKind[]) => {
+const findInTier = (tier: string[], name: string) => tier.find((elem) => elem === name);
+
+const isSelectedBy = (bs: K8sResourceKind, tier1: string[], tier2: string[]): number => {
+  if (tier1.length) {
+    const searchTier1 = findInTier(tier1, bs?.metadata?.name);
+    if (searchTier1) return 0;
+  } else if (tier2.length) {
+    const searchTier2 = findInTier(tier2, bs?.metadata?.name);
+    if (searchTier2) return 1;
+  }
+  return -1;
+};
+
+const getTableRows = (list: K8sResourceKind[], state: State) => {
```

Comment:
Add return type

Reply:
It's a complicated object I'd rather define an any[] for now(which again is implicit).